test/test_matpower.jl

```diff
@@ -0,0 +1,21 @@
+# Verify solutions against matpower results
```

Comment:
That's nice to have tests implemented for the matpower case!

Reply:
We now have confirmed power flow has same convergence pattern as matpower. Thats nice.